layout/_third-party/math/mathjax.swig

```diff
@@ -33,15 +33,19 @@
   MathJax.Hub.Queue(function() {
     var all = MathJax.Hub.getAllJax(), i;
     for (i = 0; i < all.length; i += 1) {
-      all[i].SourceElement().parentNode.className += ' has-jax';
+      document.getElementById(all[i].inputID + '-Frame').parentNode.className += ' has-jax';
     }
   });
 </script>
 <script src="{{ theme.math.mathjax.cdn }}"></script>
 
 <style>
-.MathJax_Display {
-  overflow-x: scroll;
+.has-jax {
+  overflow-x: auto;
   overflow-y: hidden;
 }
+
+li {
+  list-style-position: inside !important;
```

Comment:
That is a bug of many browser(chrome, safari, firefox ...), [See here](https://stackoverflow.com/questions/21769248/).

In short, if you put something with 'overflow' css inside a `li` element, the leading symbol (dot/circle for unordered or number for unordered) **WILL DISAPPERA**. 

You can have a simple try w/o using `list-style-position: inside !important;` the flowing content.
```tex
- $\alpha$
- $\beta$
```

Because it is a bug of browser, I didn't found any better solution. You can remove this style if necessary.

Reply:
This could cause a problem that the start indent of any list looks too long. Please check the behavior of usual list before and after setting "list-style-position: inside".

Reply:
The issue I just closed actually is also caused by it:
https://github.com/theme-next/hexo-theme-next/issues/721